crates/arroyo/src/main.rs

```diff
@@ -25,6 +25,7 @@ use tracing::{debug, error, info};
 use uuid::Uuid;
 
 #[cfg(all(
+    feature = "profiling",
```

Comment:
This enables jemalloc only when profiling is enabled, which doesn't make sense to me. Profiling only works on certain environments, which is much more restrictive than jemalloc in general.

Reply:
Sorry, I will recover later